ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/ldes/eventstream/services/EventStreamConverterImpl.java

```diff
@@ -38,15 +43,12 @@ public Model toModel(final EventStream eventStream) {
 	}
 
 	private List<Statement> addCollectionStatements(EventStream eventStream) {
-		List<Statement> statements = new ArrayList<>();
-		Resource collection = createResource(ldesConfig.getHostName() + "/" + eventStream.collection());
-		statements.add(createStatement(collection, RDF_SYNTAX_TYPE, createResource(LDES_EVENT_STREAM_URI)));
-		addStatementIfMeaningful(statements, collection, TREE_SHAPE, eventStream.shape());
-		addStatementIfMeaningful(statements, collection, LDES_VERSION_OF, eventStream.versionOfPath());
-		addStatementIfMeaningful(statements, collection, LDES_TIMESTAMP_PATH, eventStream.timestampPath());
-		eventStream.views().forEach(view -> addStatementIfMeaningful(statements, collection, TREE_VIEW,
-				ldesConfig.getHostName() + "/" + ldesConfig.getCollectionName() + view.getFragmentId()));
-		return statements;
+		String eventStreamId = ldesConfig.getHostName() + "/" + eventStream.collection();
+		List<String> views = eventStream.views().stream().map(TreeNode::getFragmentId)
+				.map(s -> ldesConfig.getHostName() + "/" + eventStream.collection() + s).toList();
+		EventStreamInfoResponse eventStreamInfoResponse = new EventStreamInfoResponse(eventStreamId,
```

Comment:
Op zich zou dit ook een static constructor kunnen zijn "EvenStreamInfoResponse.from(eventStreamId, eventStream)" want overal waar deze response initialiseren vertrekken we momenteel van een Evenstream en moet iedereen deze code dupliceren en ook deze kennis van de EventStream api hebben.